sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/IntraBundleParallelization.java

```diff
@@ -221,7 +221,7 @@ public void processElement(final ProcessContext c) throws Exception {
       }
 
       if (failure.get() != null) {
-        throw Throwables.propagate(failure.get());
```

Comment:
Prefer to deprecate and replace different functions in different commits as
for sanity

On Mon, May 2, 2016 at 1:44 PM, Thomas Groh <notifications@github.com>
wrote:

> In
> sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/IntraBundleParallelization.java
> <https://github.com/apache/incubator-beam/pull/270#discussion_r61800893>:
>
> > @@ -221,7 +221,7 @@ public void processElement(final ProcessContext c) throws Exception {
> >        }
> >
> >        if (failure.get() != null) {
> > -        throw Throwables.propagate(failure.get());
>
> Also replace the Throwables.propagateIfPossible on line 234?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-beam/pull/270/files/046feca39256399c891e5ab386a67f1a8cd8deee#r61800893>
>